modules/v2x/proto/v2x_obu_traffic_light.proto

```diff
@@ -39,9 +39,9 @@ message RoadTrafficLight {
   repeated LaneTrafficLight lane_traffic_light = 1;
   optional int32 road_direction = 2;
 }
-message OBUTrafficLightData {
+message ObuTrafficLightData {
```

Comment:
No need for `Data`. Can be renamed to `ObuTrafficLight` if that name is not already taken. Besides, it matches the filename:`v2x_obu_traffic_light.proto`.